client/src/components/app/AppHeader.js

```diff
@@ -29,6 +32,39 @@ export default function AppHeader() {
           <FaUserAlt className="itemIcon userAvatarIcon" />
         </Link>
       </div>
+
+      <Switch>
+        <Route
+          exact
+          path="/communities/new"
+          render={() => {
```

Comment:
Why don't you just put this inisde the Route? instead of using the `render` prop?

Reply:
Good point - done - looks a lot nicer ;-)